client/dashboard/sites/logs/index.tsx

```diff
@@ -71,6 +93,11 @@ function SiteLogs( { logType }: { logType: LogType } ) {
 				asOverlay
 				{ ...getLogsCalloutProps() }
 			>
+				{ autoRefreshDisabledReason && (
```

Comment:
Would it make sense to disable the auto-refresh toggle if this variable is true 🤔